adagrams/game.py

```diff
@@ -1,13 +1,129 @@
 from random import randint
 
 def draw_letters():
-    pass
+    hand_length = 10
+    draw_pool = build_draw_pool()
+    hand = []
+    for i in range(hand_length):
```

Comment:
Love that you are ensuring that the correct weights per letter are being accounted for with this loop. When we are using a for range loop that doesn't utilizes the loop variable, `i`, we can instead use `_` to signal to other developers and ourselves that this loop is just to run our iteration logic **x** amount of times. 
```suggestion
    for _ in range(hand_length):
```